action/index.js

```diff
@@ -4,26 +4,22 @@ var mqtt = require("mqtt");
 
 function main(params) {
   return new Promise((resolve, reject) => {
-    var pemFile = fs.readFileSync(params.IOT_PEM);
-    // the clientid format from the docs: d:orgId:deviceType:deviceId.
-    // const clientID = params.IOT_CLIENTID + `-${Date.now()}`;
     const options = {
       host: params.IOT_HOST,
       protocol: params.IOT_PROTOCOL,
-      username: params.IOT_USERNAME,
       port: params.IOT_SECURE_PORT,
-      password: params.IOT_PASSWORD,
       clientId: params.IOT_CLIENTID,
-      cert: pemFile,
+      username: params.IOT_DEVICE_ID,
+      password: params.IOT_PASSWORD,
     };
 
     const client = mqtt.connect(options);
 
-    client.on("connect", function (err) {
-      if (err) {
-        console.log(`error connecting ${params.IOT_DEVICE_ID}`);
-        console.log(JSON.stringify(err));
-      }
+    client.on("connect", function (packet) {
+      // if (err) {
+      //   console.log(`error connecting ${params.IOT_DEVICE_ID}`);
```

Comment:
I wonder if we should add conditional formatting with something like [pino](https://www.npmjs.com/package/pino) in combination with `process.env.NODE_ENV`? Will be good to capture and monitor logs in production?